document_merge_service/api/unoconv.py

```diff
@@ -148,7 +186,9 @@ def process(self, filename, convert):
         :return: UnoconvResult()
         """
         # unoconv MUST be running with the same python version as libreoffice
-        p = run(self.cmd + ["--format", convert, "--stdout", filename])
+        pipe = str(uuid4())
+        cmd = f"{self.cmd} --timeout 10 --pipe {pipe} --format {convert} --stdout {filename}"
+        p = run(cmd)
```

Comment:
I also incread the timeout from 3 to 10, it should help if there is a lot of load on production.